### ISSUE TITLE
Unclear doc of delta replication of ORMap

### ISSUE THREAD

**Issue by patrick-premont** [akka/akka#25738]
The documentation paragraph about delta replication within this [section](https://github.com/akka/akka/blob/master/akka-docs/src/main/paradox/distributed-data.md#maps) does not seem clear to me.

**Comment by patriknw** [comment]
yeah, that's difficult to parse. @patrick-premont could you create a PR suggesting another phrasing?

I think the essence of how it works is that deltas are only on the top level of the map, so deltas of deltas are not used.

With the exception of the `ORMultiMap.emptyWithValueDeltas`.
